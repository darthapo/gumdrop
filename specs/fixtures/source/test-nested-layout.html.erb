<% set layout:'sub' %><span>Nested.</span>