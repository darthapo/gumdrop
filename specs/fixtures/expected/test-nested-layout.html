<body><section><span>Nested.</span></section></body>